src/models/Order.js

```diff
@@ -1,7 +1,7 @@
 const mongoose = require('mongoose');
 
 // mongoose schema for the individual order items
-const orderItemSchema = new mongoose.Schema({
+const OrderItemSchema = mongoose.Schema({
```

Comment:
Was there a reason to change this from using the `new ...` constructor?

Reply:
@akosasante I thought it works without `new` too. I forgot to write it in Review.js and i guess it worked ok. just added it to both Schemas

Reply:
Ah I see, looks like you're correct it works both when invoking as a class constructor (`new`) and when calling it as a function :+1: . I just wasn't sure if you made the change for a particular reason here. But seems fine either way